misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -425,4 +428,26 @@ class FakeRedis : Redis {
   override fun zscore(key: String, member: String): Double? {
     throw NotImplementedError("Fake client not implemented for this operation")
   }
+
+  override fun zrange(
```

Comment:
Could we add some javadoc explaining why we don't implement these methods. We should nudge people to use the DockerRedis for testing if they need this.

Reply:
There is already [doc](https://github.com/cashapp/misk/blob/master/misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt#L19-L22) at the top of the class.